modules/acquia_cms_headless/tests/src/Functional/HeadlessHybridModeTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Drupal\Tests\acquia_cms_headless\Functional;
+
+/**
+ * Tests for acquia_cms_headless Hybrid mode.
+ *
+ * @group acquia_cms_headless
+ * @group low_risk
+ */
+class HeadlessHybridModeTest extends HeadlessTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'acquia_cms_toolbar',
```

Comment:
@deepakmishra2 This should be either integrated tests or we should skip this tests if module: acquia_cms_toolbar is not present or maybe we should people like other drupal.org modules i.e adding optional module dependencies in require-dev section ?
cc: @chandan-singh7929  / @rajeshreeputra

Reply:
@vishalkhode1 - we had a long discussion among team members to either make it integrated or require acquia_cms_toolbar to check for menu. So we finalized to use the module instead making it integrated test. Let me know your thoughts @chandan-singh7929  @rajeshreeputra

Reply:
If not integrated, then let's either module dependencies in dev or enable module through code and test if module exist and is enabled, so that in isolation it doesn't break. See example [here](https://github.com/acquia/acquia_cms/blob/develop/modules/acquia_cms_search/tests/src/FunctionalJavascript/AcquiaFacetSearchTest.php#L118).

Reply:
done